daemon/mgr/spec_process.go

```diff
@@ -62,21 +63,30 @@ func setupProcessTTY(ctx context.Context, c *ContainerMeta, spec *SpecWrapper) e
 }
 
 func setupProcessUser(ctx context.Context, c *ContainerMeta, spec *SpecWrapper) (err error) {
+	user := specs.User{}
```

Comment:
I would invite @Ace-Tang to take a look at the implementation in `setupProcessUser`.